requirements.txt

```diff
@@ -1,7 +1,8 @@
-django==4.1
+django==4.2
```

Comment:
Why do you need push these changes to remote? Will it not work with original versions?

Reply:
Yes, like I see it doesn`t work without this version